xmake/modules/core/tools/clang.lua

```diff
@@ -250,58 +222,59 @@ function nf_runtime(self, runtime, opt)
             }
         end
     end
-    if not self:is_plat("android") then -- we will set runtimes in android ndk toolchain
+    local target = opt.target or opt
+    -- llvm on windows still doesn't support autolinking of libc++ and compiler-rt builtins
+    -- @see https://discourse.llvm.org/t/improve-autolinking-of-compiler-rt-and-libc-on-windows-with-lld-link/71392/10
+    -- and need manual setting of libc++ headerdirectory 
+    -- @see https://github.com/llvm/llvm-project/issues/79647
+    local llvm_dirs = toolchain_utils.get_llvm_dirs(self:toolchain())
```

Comment:
Because of the global cache, even different LLVM toolchains will use the same LLVM toolchain directory.